src/classes/UTIL_AbstractCommon_BATCH.cls

```diff
@@ -262,16 +262,16 @@ public abstract inherited sharing class UTIL_AbstractCommon_BATCH implements Dat
     @testVisible private String getQueryNonLDVWhereClause() {
         String fullWhereClause = '';
 
-        String whereClause = getWhereClause();
+        String whereClause = this.getWhereClause();
 
         Boolean mainWhere = String.isNotBlank(whereClause);
-        Boolean addtlWhere = String.isNotBlank(additionalWhereClause);
+        Boolean addtlWhere = String.isNotBlank(this.additionalWhereClause);
```

Comment:
What would you think of renaming these vars as `hasMainWhereClause' and `hasAdditionalWhereClause`?